### PR TITLE
Add stop icon

### PR DESCRIPTION
**Pull request by kouknick**
### Background
Adds a new icon in the library

### Changes

- Add stop icon
### Screenshots
image

### CODE REVIEW

**Review by alexmylonas — APPROVED** [review]
LGTM!

File: src/theme/icons.tsx [hunk 1]
```diff
@@ -464,6 +464,14 @@ const icons = {
   play: {
     path: <path d="M8 5V19L19 12L8 5Z" />,
   },
+  stop: {
+    path: (
+      <path
+        transform="translate(4.000000, 4.000000)"
```
Comment (benjamminj) [inline src/theme/icons.tsx:470]: This feels strange to see in the SVG output, is this the SVG output directly from Figma or is it coming from another tool?

**Review by benjamminj — COMMENTED** [review]
One quick question about the SVG, then looks good to me!

File: src/theme/icons.tsx (hunk 1 above)
Comment (kouknick) [reply src/theme/icons.tsx:470]: No it's not directly from figma. I used an online tool to compress it.

Comment (benjamminj) [reply src/theme/icons.tsx:470]: ahhh ok no worries.

BTW, curious which online tool? I've always used https://jakearchibald.github.io/svgomg/, but always on the lookout for new ones 😄 

Can we try this `path`? This was run thru SVGOMG and I think it's even smaller ✨ 

```
<path d="M5 4h14c.5523 0 1 .44772 1 1v14c0 .5523-.4477 1-1 1H5c-.55228 0-1-.4477-1-1V5c0-.55228.44772-1 1-1Z" />
```

**Review by benjamminj — APPROVED** [review]
LGTM, thanks for answering my questions!

File: src/theme/icons.tsx (hunk 1 above)
Comment (kouknick) [reply src/theme/icons.tsx]: Thanks for the suggestion @benjamminj! super usefull. I'll use this one!

**Pull request merged**